protocol/lib/ante/unsupported_msgs.go

```diff
@@ -18,7 +18,19 @@ func IsUnsupportedMsg(msg sdk.Msg) bool {
 		// ------- CosmosSDK default modules
 		// gov
 		*govbeta.MsgSubmitProposal,
-		*gov.MsgCancelProposal:
+		*gov.MsgCancelProposal,
```

Comment:
Can you remind me the diff/relationship between this list and the list in app/msgs/normal_msgs.go? Why is this list (12 lines) shorter than the above?

Reply:
The ante/unsupported_msg.go file is the file that actually filters out unsupported messages. The msg/unsupported_msgs.go just instantiates unsupported messages

Reply:
The reason it is smaller is for 2 reason:1
1. It does not include response messsages
2. The rest of the ones not included do not have a valid message format https://github.com/dydxprotocol/v4-chain/blob/main/protocol/testutil/msgs/util.go#L50. I.e do not start with Msg